server.py

```diff
@@ -84,9 +138,18 @@ def __init__(self):
 
         # variables for game
         self.chosen_champions = [CHAMPION_IS_NOT_CHOSEN for _ in range(MAX_PLAYERS)]
+        self.players = [None, None, None]
```

Comment:
add some constans like EMPTY_SLOT and replace this None

Reply:
replaced